edb/lib/std/30-datetimefuncs.eql

```diff
@@ -21,21 +21,66 @@
 
 
 CREATE FUNCTION
-std::current_date() -> std::date
+std::datetime_now() -> std::datetime
```

Comment:
Perhaps, `datetime_current` would be a better name, making some distance from SQL `now()` which is a transaction timestamp.

Reply:
Sure, let's call it `datetime_current`. Avoiding confusion with SQL `now()` is important here.